urbanopt/geojson-modelica-translator
# RC Terminal Units and ETS Coupling

#### Any background context you want to provide?
The TEASER models generate multiple thermal zone models (between 6 and 7 based on the type of building) and need to be connected together for the whole building.

#### What does this PR accomplish?
* Use PartialBuilding model to connect the multiple thermal zones to the ideal terminal units
* Add TeaserConnector and TeaserConnectorETS 
* Add tests for both the generic coupling and ETS coupling

#### How should this be manually tested?
* Run py.test (just need the test_teaser and test_teaser_ets for now)
* Verify that models run in JModelica and Dymola

#### What are the relevant tickets?
#68 #75

#### Screenshots (if appropriate)
